Assets/MixedRealityToolkit-SDK/StandardAssets/Controllers/RiftControllers/Animations/l_controller_button01_anim.fbx.meta

```diff
@@ -0,0 +1,138 @@
+fileFormatVersion: 2
+guid: 5656ce9e2c4daca43b3b6dde03763e38
```

Comment:
The name of these controllers is a bit strange as well.

Reply:
The names are set as per the SDK they came from.

Reply:
Dang, these names suck.